.github/workflows/pytest.yaml

```diff
@@ -0,0 +1,29 @@
+name: Run Pytest
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.9
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install geograypher
```

Comment:
Is this installing from the current branch locally of the code or the version on pypi?

Reply:
Great question, it appears to be the pypi version. I'll change it to the local one